jstp/src/main/java/com/metarhia/jstp/connection/SimpleConnectionListener.java

```diff
@@ -3,7 +3,7 @@
 import com.metarhia.jstp.core.JSInterfaces.JSObject;
 
 /**
- * Created by lundibundi on 2/22/17.
+ * Simple connection listener to overwrite only methods needed by user
```

Comment:
Let's elaborate: 'Simple connection listener that overwrites all {@link ConnectionListener} methods to allow a user to only overwrite methods needed and avoid writing stubs for the other methods'.